### ISSUE TITLE
V's shoulders stretches out while walking (patch 1.20 (mod v38)

### ISSUE THREAD

**Issue by mvadam** [striderxfossility/tppmodcyberpunk#29]
Mod works great again but v's shoulders stretches really wide when walking. thank you in advanced for updating the mod to the recent patch.

**Comment by striderxfossility** [comment]
yes know issue, has to do with animations

**Comment by mvadam** [comment]
Sorry, i found out after typing that issue that it's a known problem but i did played the game before and walking around didn't have that affect at all. I just find that weird is all.

**Comment by striderxfossility** [comment]
Yes i know, i have the problem sometimes and other times it works perfectly. same as climping on to something, most of the times it does not show any issues, but sometimes these gorrilla arms pop up.

**Comment by striderxfossility** [comment]
Try wearing different clothes or a different save with other looking female, maby there is the issue
